packages/client/src/profile/index.ts

```diff
@@ -84,4 +92,51 @@ export class Profile {
       BigInt(fee),
     );
   }
+
+  public async getUsers(params: {
```

Comment:
naming just `getBy` sounds good for me, we will already be in the profile context 
so talentLayer.profile.getBy(...) is simpler and easier to understand

Reply:
renamed